client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/readclient/FlinkShuffleClientImpl.java

```diff
@@ -130,17 +135,14 @@ public FlinkShuffleClientImpl(
         new FlinkTransportClientFactory(context, conf.clientFetchMaxRetriesForEachReplica());
     this.setupMetaServiceRef(driverHost, port);
     this.driverTimestamp = driverTimestamp;
+    this.appUniqueId = appUniqueId;
```

Comment:
appUniqueId can be removed from FlinkShuffleClientImpl if  appUniqueId access is set `protected` in parent class ?

Reply:
done